lib/routes/routePUT.js

```diff
@@ -33,6 +33,42 @@ export default function routePUT(request, response, log, utapi) {
                     return routesUtils.responseNoBody(err, null, response, 200,
                         log);
                 });
+            } else if (request.query.versioning !== undefined) {
+                if (request.post === '') {
+                    log.debug('request xml is missing');
+                    return routesUtils.responseNoBody(
+                        errors.MalformedXML, null, response, null, log);
+                }
+                const xmlToParse = request.post;
+                return parseString(xmlToParse, (err, result) => {
+                    if (err) {
+                        log.debug('request xml is malformed');
+                        return routesUtils.responseNoBody(
+                            errors.MalformedXML, null, response, null, log);
+                    }
+                    const Status = result.VersioningConfiguration.Status ?
+                        result.VersioningConfiguration.Status[0] : undefined;
+                    const MfaDelete = result.VersioningConfiguration.MfaDelete ?
+                        result.VersioningConfiguration.MfaDelete[0] : undefined;
+                    const validStatuses = ['Enabled', 'Suspended'];
```

Comment:
Since those are defined once and for all, could we have them in the global scope instead of allocating them every time we go into this call? Also shouldn't this have `undefined` as well?